src/hatti/views/dataview.cljs

```diff
@@ -104,21 +105,31 @@
                            (map dataview-map) (remove nil?))
             dv->link (fn [{:keys [view label]}]
                        (let [active? (some #(= view %) (-> app-state :views :active))]
-                         (if (and (= view :map) no-geodata?)
+                         (cond
+                           (and (= view :map) no-geodata?)
                            [:a {:class "inactive" :title "No geodata"}
                             (name view)]
+                           (and (= view :map)
+                                (> (-> app-state
+                                       :dataset-info
+                                       :num_of_submissions)
+                                   *mapping-threshold*))
+                           [:a {:class "inactive"
+                                :title (str "Map does not support more than 10,000 points.")}
```

Comment:
shouldn't this take the var `mapping-threshold` as a parameter to build the error? since it's a var and all